### PR TITLE
Feat: new toggle for charts and history table

### LINKED ISSUE #2472: UI Tweak: Update chart / history settings toggles
### Description

Update chart and history toggles to the latest design, making them more consistent.

Desktop
Screenshot 2023-06-13 at 11 15 09

Tablet
image (11)

### PR DESCRIPTION
**Pull request by LeifuChen**
## Description
1. Update the chart and history toggle to the latest design, making them more consistent.
2. Adjust the position of the `HoursToggle`.

## Related issue
closes #2472 

## How Has This Been Tested?
1. Open the market page, and click on `Price` and `Funding` to see the change in the chart.
2. Open the market page, and click on `Show` and `Hide` to see the change of the history table.

## Screenshots (if appropriate):
image

### CODE REVIEW

**Review by avclarke — DISMISSED** [review]
Looks great, nice work!

File: components/Text/Body.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import styled, { css } from 'styled-components';
 export type BodyProps = React.HTMLAttributes<HTMLParagraphElement> & {
 	size?: 'xsmall' | 'small' | 'medium' | 'large';
 	weight?: 'regular' | 'bold' | 'black';
-	color?: 'primary' | 'secondary' | 'tertiary' | 'positive' | 'negative' | 'preview';
+	color?: 'primary' | 'secondary' | 'tertiary' | 'quaternary' | 'positive' | 'negative' | 'preview';
```
Comment (avclarke) [inline components/Text/Body.tsx:7]: I don't think we need a new color type for this as it's very specific, we should probably just override the body component inside the text toggle.

Comment (LeifuChen) [reply components/Text/Body.tsx]: Thanks. Updated.

File: components/TextToggle/TextToggle.tsx [hunk 2]
```diff
@@ -38,6 +34,7 @@ const TextToggle: React.FC<TextToggleProps<any>> = ({
 const ToggleButton = styled(Body)<{ $active: boolean }>`
 	cursor: pointer;
 	text-transform: capitalize;
+	color: ${(props) => props.theme.colors.selectedTheme.newTheme.text.quaternary};
```
Comment (avclarke) [inline components/TextToggle/TextToggle.tsx:37]: I think we can rename it `text.disabled` or something more relevant.

Comment (LeifuChen) [reply components/TextToggle/TextToggle.tsx]: Good idea. Updated.

**Review by avclarke — APPROVED** [review]

**Pull request merged**